source/about/structure.rst

```diff
@@ -22,6 +22,8 @@ before going into detail:
 +-------------------------------------------------------------------+-------------------------------------------------------+---------------------------------------------------------------------------------+
 | `SpongeDocs <https://github.com/Spongepowered/SpongeDocs>`_       | The official SpongeProject Documentation              | Expanding, fixing and writing the SpongeDocs                                    |
 +-------------------------------------------------------------------+-------------------------------------------------------+---------------------------------------------------------------------------------+
+| `SpongeHome <https://github.com/Spongepowered/SpongeHome>`_       | The brand new website for the SpongeProject           | Development of our website                                                      |
```

Comment:
Don't write in the present text here.

Give it a month or 2 after this gets merged and it will need editing again to remove "brand new" Best to just do it now and not need an edit later.